src/crate/crash/repl.py

```diff
@@ -222,7 +222,7 @@ def is_multiline():
 
 class Capitalizer:
 
-    KEYWORD_RE = r'(?:"\w+)|(?:\'\w+)|\w+'
+    KEYWORD_RE = r'(?:"\w+)|(?:\'\w+)|(?:\\\w+)|\w+'
```

Comment:
this does not catch the case `\format dynamic`

You need to disable auto-captialisation when line starts with `\`